### PR TITLE
Fixed one typo and made one sentence more readable by adding 'and' to it.

### PR DESCRIPTION
**Pull request by anttimo**

### CODE REVIEW

File: docs/DirectManipulation.md [hunk 1]
```diff
@@ -155,11 +155,11 @@ You may have noticed that we passed all of the props down to the child
 view using `{...this.props}`. The reason for this is that
 `TouchableOpacity` is actually a composite component, and so in addition
 to depending on `setNativeProps` on its child, it also requires that the
-child perform touch handling. To dot this, it passes on [various
+child perform touch handling. To do this, it passes on [various
 props](https://facebook.github.io/react-native/docs/view.html#onmoveshouldsetresponder)
 that call back to the `TouchableOpacity` component.
 `TouchableHighlight`, in contrast, is backed by a native view only
```
Comment (ide) [inline docs/DirectManipulation.md]: think this was supposed to say "native view [and] only requires"

Comment (anttimo) [inline docs/DirectManipulation.md]: True, I will change it.

**Comment by ide** [comment]
oh and could you please squash your commits (git rebase -i) for documentation PRs?

**Comment by anttimo** [comment]
Sure thing. Actually read the documentation about it but already forgot it when doing the additional commit.

**Comment by ide** [comment]
Thanks!

**Pull request merged**